jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -5978,7 +5984,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\"webappId\":{{AppId}},\"name\":\"HEALTH-CHECK-4.0\",\"webappContext\":\"/hello\",\"groupId\":{{GroupId}},\"loadBalanceAcrossServers\":true}"
+							"raw": "{\"webappId\":{{AppId}},\"name\":\"HEALTH-CHECK-4.0.1\",\"webappContext\":\"/hello\",\"groupId\":{{GroupId}},\"loadBalanceAcrossServers\":true}"
```

Comment:
Don't hard code this group name.
Also this looks CTP health check specific and should be removed.